.travis/mypy-stubs/libzfs.pyi

```diff
@@ -336,53 +336,3 @@ class ZfsConverter:
     __init__ = ... # type: Any
     to_native = ... # type: Any
     to_property = ... # type: Any
-
-class datetime(date):
```

Comment:
oh?

Reply:
It was never used in our context, but caused confusion. That's why I have removed it. Do you think that was an error?